diff --git a/LDAPapi.pm b/LDAPapi.pm
index ce5de49..8fa91ce 100644
--- a/LDAPapi.pm
+++ b/LDAPapi.pm
@@ -232,7 +232,7 @@ require AutoLoader;
        LDAP_TAG_IM_RES_VALUE
        LDAP_TAG_SASL_RES_CREDS
        );
-$VERSION = '3.0.x';
+$VERSION = '3.0.5';
 
 sub AUTOLOAD {
     # This AUTOLOAD is used to 'autoload' constants from the constant()
diff --git a/README b/README
index f7edf11..6229e0f 100644
--- a/README
+++ b/README
@@ -1,5 +1,5 @@
 =======================================================================
-                Net::LDAPapi Module v3.0.x for Perl5
+                Net::LDAPapi Module v3.0.5 for Perl5
 		by Quanah Gibson-Mount @ Symas, Inc.
 	based on version 2.00 by Quanah Gibson-Mount @ Stanford University
 	based on version 1.50 by Howard Chu @ Symas Corporation
